custom-elements.json

```diff
@@ -169,8 +169,8 @@
       "attributes": [
         {
           "name": "events",
-          "description": "array containg events from user agenda.",
-          "jsDoc": "/**\n   * array containg events from user agenda.\n   * @type {Array<MicrosoftGraph.Event>}\n   */",
+          "description": "array containing events from user agenda.",
```

Comment:
Not sure, but this description might be coming from the component source and might make more sense to update it there.

Reply:
Yeah this will need to be changed from source comments in the ts files.